web-marketplace/src/pages/BuyCredits/BuyCredits.tsx

```diff
@@ -0,0 +1,89 @@
+import { useMemo, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { useLingui } from '@lingui/react';
+
+import { useFetchSellOrders } from 'features/marketplace/BuySellOrderFlow/hooks/useFetchSellOrders';
+import WithLoader from 'components/atoms/WithLoader';
+import { CardSellOrder } from 'components/organisms/ChooseCreditsForm/ChooseCreditsForm.types';
+import { MultiStepTemplate } from 'components/templates/MultiStepTemplate';
+import { useGetProject } from 'components/templates/ProjectDetails/hooks/useGetProject';
+
+import { PAYMENT_OPTIONS } from './BuyCredits.constants';
+import { BuyCreditsForm } from './BuyCredits.Form';
+import { PaymentOptionsType } from './BuyCredits.types';
+import { getFormModel } from './BuyCredits.utils';
+
+export const BuyCredits = () => {
+  const { _ } = useLingui();
+  const {
+    sanityProject,
+    projectBySlug,
+    loadingProjectBySlug,
+    projectByOnChainId,
+    loadingProjectByOnChainId,
+    offchainProjectByIdData,
+    loadingOffchainProjectById,
+    isBuyFlowDisabled,
+    projectsWithOrderData,
+    onChainProjectId,
+    offChainProject,
+    creditClassOnChain,
+    loadingBuySellOrders,
+  } = useGetProject();
```

Comment:
We can remove unused properties to only include the properties that are actually used in the component

```
  const {
    sanityProject,
    isBuyFlowDisabled,
    projectsWithOrderData,
    onChainProjectId,
    offChainProject,
    creditClassOnChain,
    loadingBuySellOrders,
  } = useGetProject();
```

Reply:
same as above, this is still WIP so I'm leaving those variables for now, because they might be used later on for the next steps